content/hardware/05.nicla/boards/nicla-vision/tutorials/image-classification/content.md

```diff
@@ -228,3 +228,9 @@ while(True):
 ## Conclusion
 
 You have learned about classification as a machine learning concept which categorizes a set of data into classes. You have also learned how supervised learning works and what quantization of a model means. Furthermore you have learned to train a custom TFLite machine learning model and deploy it to your board.
+
+## Troubleshooting
+
+### GitHub Workflow
+
+If you’re encountering difficulties while using the GitHub workflow in your forked repository, make sure that your `MobileNetV2 96x96 0.1` as model type, otherwise the model will likely be too big and not work. Also make sure you are changing the files in your forked repository and not in the original OpenMV repository.
```

Comment:
```suggestion
If you’re encountering difficulties while using the GitHub workflow in your forked repository, make sure that you chose `MobileNetV2 96x96 0.1` as model type, otherwise the model will likely be too big. Also make sure you are changing the files in your own, forked repository and not in the original OpenMV repository.
```